examples/pytorch/translation/run_translation.py

```diff
@@ -243,16 +244,17 @@ def main():
         datefmt="%m/%d/%Y %H:%M:%S",
         handlers=[logging.StreamHandler(sys.stdout)],
     )
-    logger.setLevel(logging.INFO if training_args.should_log else logging.WARN)
+
+    log_level = training_args.get_node_log_level()
+    logger.setLevel(log_level)
+    datasets_module.utils.logging.set_verbosity(log_level)
```

Comment:
This is not nice, we should make a first issue to have all the instance of `datasets` objects be switched to `raw_datasets` so we can use the proper module name (separate from this PR).

Reply:
I will do it in this PR if you don't mind, so it's be easier to replicate the changes to other examples.

Reply:
Yes yes, I wasn't saying this had to be done here, just pointing out the need :-)

Reply:
we are on the same page, I was the one who wanted to do it here and was asking if it was ok.